### ISSUE TITLE
no such file or directory error in deps.compile

### ISSUE THREAD

**Issue by shahryarjb** [IvanRublev/Domo#29]
### Environment

* Elixir version (elixir -v): 1.15
* Erlang: OTP 26
* Domo version (mix deps | grep domo | head -1): [1.5.12](https://hex.pm/packages/domo/1.5.12)

### Actual behavior

When I want to compile deps, it shows me this error
```
Domo makes type ensures for standard lib modules Date, Date.Range, DateTime, File.Stat, File.Stream, GenEvent.Stream, IO.Stream, Macro.Env, NaiveDateTime, Range, Regex, Task, Time, URI, Version.
Domo is compiling type ensurer for 15 modules (.ex)

== Compilation error in file lib/compile_stdlib_type_ensurers.ex ==
** (File.Error) could not write to file "/Users/shahryar/Documents/Programming/Elixir/mishka_pub/_build/dev/lib/domo/.mix/resolved_stdlib_types.domo": no such file or directory
    (elixir 1.15.0) lib/file.ex:1117: File.write!/3
    lib/compile_stdlib_type_ensurers.ex:30: (file)
    (elixir 1.15.0) lib/kernel/parallel_compiler.ex:377: anonymous fn/5 in Kernel.ParallelCompiler.spawn_workers/8
could not compile dependency :domo, "mix compile" failed. Errors may have been logged above. You can recompile this dependency with "mix deps.compile domo --force", update it with "mix deps.update domo" or clean it with "mix deps.clean domo"
```

**Comment by IvanRublev** [comment]
Hi, that's an interesting one.

The file is not in place, which should be. Try to clean and recompile the project with `mix clean --deps && mix deps.get && mix compile`. If the problem still exists, please, add a small project that reproduces the problem to this issue, and I'll have a look.

**Comment by shahryarjb** [comment]
Hello, I uploaded a simple project:

https://github.com/shahryarjb/testproject

I think it should manage permission of the directory and files, but I think 

Thank you in advance

**Comment by shahryarjb** [comment]
Hello Dear @IvanRublev,  I run `mix test` on Elixir 1.15 and OTP 26, because is not there OTP26 on your GitHub action

# Elixir 1.15 and OTP26 on you new branch
```
430 tests, 240 failures, 35 invalid
```

Just for information and pleas if is possible add OTP26 for your new branch to test.

I am sorry for asking and thank you for all your efforts, is there a deadline to fix these errors for new version of elixir?

By the way, the `dialyxir` should be last one version to have no error for elixir 1.15 because for [erlex](https://hex.pm/packages/erlex)

Thank you in advance

**Comment by IvanRublev** [comment]
Hey,
You don't need to be sorry.
Thanks for your input. Dialyxir is the thing I missed! 
Please, stay tuned. An update will be today 😉

**Comment by IvanRublev** [comment]
Hey @shahryarjb,

I've released the version https://hex.pm/packages/domo/1.5.13 that fixes the bug it supports Elixir v1.15 and OTP 26. Please, have a look and if the problem persists reopen this issue.

**Comment by shahryarjb** [comment]
Thank you @IvanRublev, 🌹

it works in compile time, but I will test the whole lib in my project, if is there problem; I create a new issue

Thank again for all your efforts 🙏🏻
